Sources/MessagesLayoutDelegate.swift

```diff
@@ -57,7 +57,7 @@ public extension MessagesLayoutDelegate {
     }
 
     func avatarSize(for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> CGSize {
-        return CGSize(width: 30, height: 30)
+        return CGSize(width: 40, height: 40)
```

Comment:
Any reason why this can't default to `30 x 30`? If it can't support this then it's not a solution :( Not trying to be hyper critical, I appreciate you working on this 👍

Reply:
So, when its 30x30 I thin MD or MG doesn't fit. That was the initial problem for the issue

Reply:
So, I think we're looking at the problem the wrong way. It's not the size of the view that's the problem (although it does need to be dynamic compared to the previous 30x30 default) -- the problem lies in the size of the font.

Reply:
I guess, it makes sense to make the font smaller in that case. I will change the default value to a smaller value.

Reply:
@gungorbasa Still not flexible enough :P If we were using a `UILabel` we could specify a `minimumFontSize` to scale down to. We can't do that here because we have a `UIImage`. We probably need to recreate an API similar to this. We can use something like `String.boundingRect(size:options:context)` to check how big of a rect we need to fit the text. Then if the avatarView is too small we shrink the font size until it fits

Reply:
@SD10 I was thinking something like that earlier, but then I realized that it might be kind of overkill. I don't know. Wouldn't it hurt the performance?

Reply:
I mean I can implement something like that, it is not a problem but that was my initial thought.

Reply:
@gungorbasa Nope, drop in the bucket compared to other things we can improve. This is how the size of every message label is calculated 😃 Anything we can do to improve performance in the implementation itself is great, but the idea itself is fine

Reply:
We could even add a boolean property that opts in to this behavior, just like UILabel:
https://developer.apple.com/documentation/uikit/uilabel/1620546-adjustsfontsizetofitwidth

Reply:
@SD10 Ok, let me try that then 😄